src/handlers/RF24StreamLogHandler.cpp

```diff
@@ -141,73 +151,47 @@ void RF24StreamLogHandler::appendFormat(const char format, va_list *args)
 void RF24StreamLogHandler::appendTimestamp()
 {
     char c[12];
-    sprintf(c, "%10lu ", millis());
+    sprintf(c, "%10lu", millis());
     stream->print(c);
-    stream->print(" ");
 }
 
 void RF24StreamLogHandler::appendLogLevel(uint8_t logLevel)
 {
-    uint8_t logMainLevel = logLevel & 0xF8;
     uint8_t subLevel = logLevel & 0x07;
 
-    switch (logMainLevel)
+    if (logLevel >= RF24LogLevel::ERROR && logLevel <= RF24LogLevel::DEBUG + 7)
     {
-        case RF24LogLevel::ERROR:
-        {
-            stream->print(rf24LogLevels[0]);
-            break;
-        }
-        case RF24LogLevel::WARN:
-        {
-            stream->print(rf24LogLevels[1]);
-            break;
-        }
-        case RF24LogLevel::INFO:
-        {
-            stream->print(rf24LogLevels[2]);
-            break;
-        }
-        case RF24LogLevel::DEBUG:
+        uint8_t logIndex = ((logLevel & 0x38) >> 3) - 1;
+        stream->print(rf24LogLevels[logIndex]);
+
+        if(subLevel == 0)
         {
-            stream->print(rf24LogLevels[3]);
-            break;
-        }
-        default:
+            stream->print("  ");
+        } else
         {
-            // unknown
-            if (logLevel < 10)
-            {
-                stream->print("Lvl   ");
-            }
-            else if (logLevel < 100)
-            {
-                stream->print("Lvl  ");
-            }
-            else
-            {
-                stream->print("Lvl ");
-            }
-            stream->print(logLevel);
-            stream->print(" ");
-            return;
+            stream->print(":");
+            stream->print(subLevel);

```

Comment:
it might be easier to print this as a OCT int using `print(subLevel, OCT);` which might also eliminate the need to add spaces in the level descriptions.

Reply:
nevermind, this will only ever be 1-7